clodius/cli/aggregate.py

```diff
@@ -1618,39 +1572,37 @@ def geojson(
     metavar='FILEPATH'
 )
 @click.option(
-    '--output-file',
     '-o',
+    '--output-file',
     default=None,
     help="The default output file name to use. If this isn't"
-    "specified, clodius will replace the current extension"
-    "with .hitile"
+         "specified, clodius will replace the current extension"
+         "with .gjdb"
 )
 @click.option(
-    '--assembly',
-    '-a',
-    help='The genome assembly that this file was created against',
-    type=click.Choice(nc.available_chromsizes()),
-    default='hg19')
-@click.option(
-    '--tile-size',
-    '-t',
-    default=256,
-    help="The number of data points in each tile."
-    "Used to determine the number of zoom levels"
-    "to create.")
-@click.option(
-    '--chromsizes-filename',
-    help="A file containing chromosome sizes and order",
-    default=None)
+    '-m',
+    '--max-per-tile',
+    default=20,
+    type=int
+)
 @click.option(
-    '--starting-resolution',
     '-s',
+    '--tile-size',
```

Comment:
I think you moved `--starting-resolution / -s` up... should the short option here either be dropped, or changed to `-t`?

Reply:
Probably dropped. I'll make the change this evening.

Reply:
Removed that option.